ApiBundle/EventSubscriber/AbstractSubscriber.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\EventSubscriber;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Symfony\Component\HttpKernel\Controller\ControllerResolverInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\KernelEvent;
+
+/**
+ * Class AbstractSubscriber
+ */
+abstract class AbstractSubscriber
+{
+    protected $resolver;
+    protected $annotationReader;
+
+    /**
+     * @param AnnotationReader            $annotationReader
+     * @param ControllerResolverInterface $resolver
+     */
+    public function __construct(AnnotationReader $annotationReader, ControllerResolverInterface $resolver)
+    {
+        $this->resolver = $resolver;
+        $this->annotationReader = $annotationReader;
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return boolean
+     */
+    protected function isApiRequest(Request $request)
+    {
+        return 0 === strpos($request->get('_route'), 'open_orchestra_api');
```

Comment:
I agree with you, but this discussion should be more wide than just on this pr.

I will add a ticket for the ceremonie of tomorrow, is it ok for you?